src/app/profile/cleanup/cleanup.component.html

```diff
@@ -58,7 +58,7 @@ <h3 *ngIf="cleanupStatus !== 'notstarted'">Results</h3>
             <template #itemTemplate let-item="item" let-index="index">
               <div class="list-pf-content-wrapper">
                 <div class="list-pf-main-content">
-                  <span class="cleanup-status-icon {{item.statusIcon}}"></span>
+                  <span *ngIf="item.statusIcon" class="cleanup-status-icon {{item.statusIcon}}"></span>
```

Comment:
The space name appears to be indented 1 extra space and therefore doesn't align with the username above.